src/ansys/fluent/core/utils/dictionary_operations.py

```diff
@@ -0,0 +1,6 @@
+"""Performs some operations on python dictionaries."""
+
+
+def get_first_dict_key_for_value(input_dict, value):
+    """Get the first dictionary key that matches a value."""
+    return next(key for key, val in input_dict.items() if val == value)
```

Comment:
`next` takes a second argument for defualt. We can return None when the value is not found, then we can skip checking for conatinment in the caller code.

Reply:
Done, Thank you.